### ISSUE TITLE
[Feature request] Add a --list-suites parameter

### ISSUE THREAD

**Issue by alexislefebvre** [sebastianbergmann/phpunit#2077]
We can list groups:

```
$ phpunit --help | grep suite
  --testsuite <pattern>     Filter which testsuite to run.
alexis@ubuntu ~/p/LiipFunctionalTestBundle (master)> phpunit --help | grep group
  --group ...               Only runs tests from the specified group(s).
  --exclude-group ...       Exclude tests from the specified group(s).
  --list-groups             List available test groups.

```

But there is no equivalent for suites:

```
$ phpunit --help | grep suite
  --testsuite <pattern>     Filter which testsuite to run.
```

What about adding a ` --list-suites` parameter?

It would useful for example to launch test suites with `parallel` command, e.g.:

```
phpunit  --list-suites | parallel --no-notice --gnu --keep-order \
'echo "Running {} tests"; phpunit --colors=always --testsuite {} ; \
or (echo -e "\e[41mFAILED\e[0m {}" && $(exit 1));'
```

**Comment by alexislefebvre** [comment]
This is extremely easy to do in Python:

```
#!/usr/bin/python

import sys, xml.etree.ElementTree as ET

root = ET.parse(sys.argv[1]).getroot()

for suite in root.iter('testsuite'):
    print suite.get('name')
```

So you can close this issue if you want since there is a workaround.

**Comment by alexislefebvre** [comment]
This has been fixed in #2203. :tada:
